tests/test_specs.py

```diff
@@ -43,6 +43,7 @@ def test_validate_spec(spec: str) -> None:
                     directory_valid,
                     json_output=None,
                     abort_on_error=False,
+                    coverage_info=None,
```

Comment:
The coverage calculation should be enabled for this test.

Reply:
Done in f8ffc5d3.